client/src/components/AddPlayer.js

```diff
@@ -1,7 +1,65 @@
-import React from 'react';
+import React, { useContext, useRef} from 'react';
+import { GameContext } from '../GameContext';
+import '../style/AddPlayer.css';
 
 const AddPlayer = () => {
-  return <div>Add some players lol</div>;
+
+  const { players, setPlayers } = useContext(GameContext);
+  const toAdd = useRef(null);
+
+  const handleRemoval = (e) => {
+    e.preventDefault();
+    let playerid = e.target.getAttribute("playerid");
+    
+    let newPlayers = [...players];
+    setPlayers( newPlayers.filter ( currentPlayers => {
```

Comment:
Did you install the Prettier extension? It'll fix your spacing automatically for you